core/src/test/kotlin/com/karumi/kotlinsnapshot/core/KotlinSerializationTest.kt

```diff
@@ -185,7 +212,7 @@ class KotlinSerializationTest {
     fun `should serialize Date object with time and timezone`() {
         val dateFormat = SimpleDateFormat("yyyy-MM-dd:HH:mm:ss.SSS", Locale.US)
         dateFormat.timeZone = TimeZone.getTimeZone("GMT")
-        val date = dateFormat.parse("2007-12-03:18:46:19.333")
+        val date = dateFormat.parse("2007-12-03:18:46:19.111")
```

Comment:
I'm curious about this change, why did you change it?

Reply:
I think it was just a random test.